### DISCUSSION TITLE
Find most retried tests with allure

### DISCUSSION THREAD

**Discussion by Diokuz** [allure-framework/allure2#1733] [TestOps]
Hi there!

We have a lot of tests, and part of them are flaky. Our launches 100% green, but inside launch there could be tests which retried several times.

Is that possible find least stable test in allure?

For now we have our custom solution, which includes database, grafana and reporter. But we are looking for allure as a possibility to give up that solution.

<img width="791" alt="Снимок экрана 2022-07-25 в 17 19 52" src="https://user-images.githubusercontent.com/2929442/180799455-40b7b3d7-e8fa-49b4-87ba-6807d2ed1a7c.png">

**Comment by cheshi-mantu** [comment]
Hi, you need to create a custom dashboard and add the following widget: 

<img width="331" alt="image" src="https://user-images.githubusercontent.com/8607348/185469637-563ddb37-6844-4669-abcc-754236293346.png">

that'll result into the following:
<img width="677" alt="image" src="https://user-images.githubusercontent.com/8607348/185469783-a4f6f533-c711-4e9c-8aa0-3649ff0c68bc.png">

to get faster responses please consider asking these either in the appropriate section (Allure TestOps or using the dedicated support portal for Allure TestOps users: https://help.qameta.io, you need to register with the corporate email address only)
